src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -365,6 +365,18 @@ const supplierRequisitionInitialiser = requisition => {
       ? sortedData
       : sortedData.filter(item => item.isLessThanThresholdMOS);
 
+  const indicators = program?.indicators?.slice() || [];
+  const [selectedIndicator] = indicators;
```

Comment:
this only works on actual arrays, I believe, whereas this *I think* is a `Realm.Results`, which is actually an object with the shape `{ 0, 1, 2 }`

Reply:
That's the idea for the `.slice()` here, I believe that returns a JS array?

Reply:
haha whoops

Reply:
![cookie](https://user-images.githubusercontent.com/43223637/71786355-6ad3f400-306f-11ea-8f90-43a74c4fe548.gif)